src/test/java/org/opensearch/ad/model/AnomalyResultBucketTests.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.ad.model;
+
+import java.io.IOException;
+
+import org.opensearch.ad.AbstractADTest;
+import org.opensearch.ad.TestHelpers;
+import org.opensearch.common.io.stream.BytesStreamOutput;
+import org.opensearch.common.io.stream.StreamInput;
+
+public class AnomalyResultBucketTests extends AbstractADTest {
```

Comment:
Could you add a test for the toXContent() method?

Reply:
Sure, will add.